### DISCUSSION TITLE
Ag Damage, Need to Accommodate Late Planting THEN Impacted.

### DISCUSSION THREAD

**Discussion by Tich1212** [USACE/go-consequences#83] [Ideas]
Probably need to return Late costs if conditions are met here:
https://github.com/USACE/go-consequences/blob/0156c787c7c19bd79b67ed924617eacabc41b29e/crops/productionfunctions.go#L22-L28

**Comment by HenryGeorgist** [comment]
this will only happen within the context of multiple events in a given season. It seems like the basic compute is a single event compute.

I think we would have to have a multi year or multi event compute capability added to support this functionality. Maybe we target that for a future milestone or future release
